var/spack/repos/builtin/packages/py-types-pyyaml/package.py

```diff
@@ -10,7 +10,7 @@ class PyTypesPyyaml(PythonPackage):
     """Typing stubs for PyYAML."""
 
     homepage = "https://github.com/python/typeshed"
-    url      = "https://files.pythonhosted.org/packages/3c/96/22a02c905aec42d8f7f7c48b7a00d68019edd4768e3fb94b754966d07870/types-PyYAML-5.4.6.tar.gz"
+    url      = "https://pypi.io/packages/source/t/types-pyyaml/types-PyYAML-5.4.6.tar.gz"
```

Comment:
In the files in the previous PR https://github.com/BlueBrain/spack/pull/1268 there are similar hashed urls in
- `var/spack/repos/builtin/packages/py-atlalign/package.py`
- `var/spack/repos/builtin/packages/py-imgaug/package.py`

Maybe we can update them as well?
However, I see that there are even more hashed urls that were added in https://github.com/BlueBrain/spack/commit/369b06727942a3e0d7b6cd3e8566668f197019c0, so maybe we can keep them as they are until a new version is needed.

Reply:
Postponing then these fixes/cleanup to the next releases.